shuup/front/views/upload.py

```diff
@@ -12,12 +12,11 @@
 from django.core.exceptions import ValidationError
 from django.http.response import HttpResponseForbidden, JsonResponse
 from django.utils.translation import ugettext as _
-from filer.models import Folder
 
+from shuup.admin.modules.media.views import get_or_create_folder
```

Comment:
Shuup front shouldn't know anything about admin module I think.

Reply:
But can front know about `shuup.utils` as it already dose?